rclpy/rclpy/impl/__init__.py

```diff
@@ -26,6 +26,18 @@ def _import(name):
         with add_dll_directories_from_env('PATH'):
             return importlib.import_module(name, package='rclpy')
     except ImportError as e:
+        if e.path is None:
+            import sysconfig
+            expected_path = os.path.join(
+                os.path.dirname(os.path.dirname(__file__)),
+                name[1:] + sysconfig.get_config_var('EXT_SUFFIX'))
+            assert not os.path.isfile(expected_path)
+            e.msg += \
+                "\nThe C extension '%s' isn't present on the system." \
+                " Please refer to '%s' for possible solutions" % (
+                    expected_path,
+                    'https://index.ros.org/doc/ros2/Troubleshooting/Installation-Troubleshooting/'
+                    '#import-failing-without-library-present-on-the-system')
```

Comment:
nit: prefer using f-strings

Reply:
Done in e76dc55bbc3de0b0201aea4994ee1bf0fb791be1.